azure-pipelines-CI.yml

```diff
@@ -126,13 +126,17 @@ extends:
               - name: _BuildArgs
                 value: /p:SignType=$(_SignType)
             steps:
+            - task: UseDotNet@2
```

Comment:
We might want to put this in the linux leg below as well. Would be easier if it was part of a common task list we can stick in all the jobs for all the other OSs.